examples/browser/camera/simple-es5-player.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Example video streaming from Axis camera</title>
+  </head>
+
+  <body>
+    <div>
+      <div>
+        <label for="device">Device IP:</label>
+        <input id="device" type="text" placeholder="192.168.0.90" />
+      </div>
+      <div>
+        Video encoding:
+        <input type="radio" name="encoding" id="h264" checked />
+        <label for="h264">H.264</label>
+        <input type="radio" name="encoding" id="jpeg" />
+        <label for="jpeg">Motion JPEG</label>
+      </div>
+      <button id="play-button">Play</button>
+    </div>
+    <div style="position: fixed; width: 1280px; height: 720px">
+      <video
+        style="position: absolute; width: 100%; height: 100%;"
+        autoplay
+      ></video>
+      <canvas style="position: absolute; width: 100%; height: 100%;"></canvas>
+    </div>
+    <script src="../media-stream-library.min.js"></script>
+	  <script src="rtsp-player.js"></script>
```

Comment:
Why is it importing the `rtsp-player.js` script instead of the `simple-es5-player.js` script featured in the PR?